modules/services/parcellite.nix

```diff
@@ -34,10 +34,9 @@ in
 
       Service = {
         # PATH have been added in nixpkgs.parcellite, keeping it here for
-        # backward compatibility. XDG_DATA_DIRS is necessary to make it pick up
-        # icons correctly.
+        # backward compatibility.
         Environment = ''
-          PATH=${package}/bin:${pkgs.which}/bin:${pkgs.xdotool}/bin XDG_DATA_DIRS=${pkgs.hicolor_icon_theme}/share
+          PATH=${package}/bin:${pkgs.which}/bin:${pkgs.xdotool}/bin
```

Comment:
I believe you could remove the `PATH` variable as well since it is set in the `parcellite` wrapper script, even on 19.09.